agave_app/agaveGui.cpp

```diff
@@ -37,12 +37,52 @@
 
 #include <filesystem>
 
+const QString darkStyleSheet = R"(

```

Comment:
What does the `R` syntax here do?

Reply:
R lets you do a inline multiline string literal without having to escape any characters.

Reply:
is there a way this still gets linted to avoid typos?

Reply:
sadly, no.  There might be a future PR where the stylesheet gets much bigger and lives in its own resource file and then there will at least be a chance - but it's a special Qt flavor of CSS and probably wouldn't lint like real css.  Not sure if Qt provides any tools for this.